### ISSUE TITLE
Nier: Automata crashes at the end of route A

### ISSUE THREAD

**Issue by Darkbytex** [Kaldaien/FAR#136]
Hello, I've been looking around the internet for other people with the same issue that I have but I haven't seen anyone have the specific crash that I am having.

Normally, the game ran great for the most part without any issues until the I've reached Eve. Whenever I defeat Eve completely in Route A and go to save my progress the game immediately crashes, I have tried every fix that I saw but none of it worked.

Note that I have never once had any crashes or stutters or any technical difficulties with this game, I've also tried playing my save file on multiple versions of the game and each of those versions the game crashed, so I am not sure if this issue originated from the save file or not. I used to play the game pirated before because I could not afford the game or have any sort of payment method, but now I have the legitimate version of the game and not the pirated version.

[modules.log](https://github.com/Kaldaien/FAR/files/2303447/modules.log)
[ReShade64.log](https://github.com/Kaldaien/FAR/files/2303448/ReShade64.log)
[steam_api.log](https://github.com/Kaldaien/FAR/files/2303449/steam_api.log)
[CEGUI.log](https://github.com/Kaldaien/FAR/files/2303450/CEGUI.log)
[crash.log](https://github.com/Kaldaien/FAR/files/2303451/crash.log)
[dinput8.log](https://github.com/Kaldaien/FAR/files/2303452/dinput8.log)
[dxgi_budget.log](https://github.com/Kaldaien/FAR/files/2303453/dxgi_budget.log)
[game_output.log](https://github.com/Kaldaien/FAR/files/2303454/game_output.log)

I would appreciate the help very much, thanks in advance.

**Comment by Idearum** [comment]
1. Verify game files through Steam.

2. Rename dinput8.dll to something else and see if the issue still occurs even without FAR. If it does, then FAR is not involved in the issue.

**Comment by Darkbytex** [comment]
I've already known it was not from FAR, I just hoped by installing FAR it would fix the issue but it appears not, at least it has given me a crash report, not sure if that will help or not I've been trying to fix this problem for a long time now and I really don't want to start a new playthrough just to see if the issue was from the save file or not. :/

If you have some insight that could maybe lead to fixing this problem I would really appreciate it, thanks again for replying.

**Comment by Idearum** [comment]
There's sadly nothing concrete in those logs. Only things I can see is that Nvidia Ansel, Start10, and Fences is hooking the game. You might want to try to disabling those, although I somewhat doubt it'll fix it.

If the issue occurs >exactly< when you try to save the game, it's possible that a cloud storage provider and/or Windows 10's "controlled folder access" is crashing the game, per https://pcgamingwiki.com/wiki/Nier:_Automata#Saving_does_not_work

**Comment by Darkbytex** [comment]
The thing is, I've played the game before any of these programs and I've also been successfully saving the game countless times and hadn't have any issues at all, as for cloud storage provider such as OneDrive that comes with windows 10, I've disabled that which caused many issues with other games before.

I've disabled everything and tried to run the game, same issue. I think something went wrong with the save file that broke it that's what I think. I don't really know what else to say :/

Thank you for sticking by Idearum but I think this problem is way too advanced. :)
